src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -44,6 +44,27 @@ public IOperation Create(BoundNode boundNode)
 
```

Comment:
>!IsImplicitConversion(boundNode as BoundConversion) [](start = 16, length = 51)

Given the issue discovered around the syntax associated with implicit conversions, perhaps we should let the conversions through here and specially handle conversion operands instead, i.e. not adding ParenthesizedExpression on operands.

Reply:
@AlekseyTs - Done.